src/test/java/com/google/api/codegen/testdata/discoveries/java/java_logging.v2beta1.json.baseline

```diff
@@ -1,871 +1,966 @@
  *    https://developers.google.com/resources/api-libraries/download/logging/v2beta1/java
+ * 3. This sample uses Application Default Credentials for authentication.
+ *    If not already done, install the gcloud CLI from
+ *    https://cloud.google.com/sdk and run
+ *    `gcloud beta auth application-default login`
  */
-
 import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
 import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
 import com.google.api.client.http.HttpTransport;
 import com.google.api.client.json.JsonFactory;
 import com.google.api.client.json.jackson2.JacksonFactory;
 import com.google.api.services.logging.v2beta1.Logging;
+import com.google.api.services.logging.v2beta1.Logging.Entries.List;
 import com.google.api.services.logging.v2beta1.model.ListLogEntriesRequest;
 import com.google.api.services.logging.v2beta1.model.ListLogEntriesResponse;
 import com.google.api.services.logging.v2beta1.model.LogEntry;
 import java.io.IOException;
 import java.security.GeneralSecurityException;
-import java.util.Collections;
-
+import java.util.Arrays;
 
 public class LoggingExample {
-  public static void main(String[] args) throws IOException, GeneralSecurityException {
-    // Use Application Default Credentials for authentication when running locally.
-    // For more information, see:
-    // https://developers.google.com/identity/protocols/application-default-credentials
-    GoogleCredential credential = GoogleCredential.getApplicationDefault();
-    if (credential.createScopedRequired()) {
-      credential = credential.createScoped(Collections.singletonList("https://www.googleapis.com/auth/cloud-platform"));
-    }
-
-    HttpTransport httpTransport = GoogleNetHttpTransport.newTrustedTransport();
-    JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
-
-    Logging loggingService = new Logging.Builder(httpTransport, jsonFactory, credential)
-      .setApplicationName("Google Cloud Platform Sample")
-      .build();
-
-
+  public void run() throws IOException, GeneralSecurityException {
+    // TODO: Assign values to the desired fields of `requestBody`.
     ListLogEntriesRequest requestBody = new ListLogEntriesRequest();
-    // TODO: Assign values to desired fields of the `requestBody` object.
 
+    Logging service = createService();
+    List request = service.entries().list(requestBody);
 
-    Logging.Entries.List request = loggingService.entries().list(requestBody);
     ListLogEntriesResponse response;
     do {
       response = request.execute();
-      if (response.getEntries() == null)
+      if (response.getEntries() == null) {
         continue;
-
-      for (LogEntry logEntry : response.getEntries()) {
-        // TODO: Change code below to process each `logEntry` resource:
-        System.out.println(logEntry);
       }
-
-      requestBody.setPageToken(response.getNextPageToken());
+      for (LogEntry item : response.getEntries()) {
```

Comment:
Why is this showing up as a new change, when you already merged #510? Please rebase on top of head; otherwise reviewing this is unnecessarily confusing and time-consuming.